<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>18/6/0</td>
</tr><tr>
<td>Started on:</td><td>Fri Jul 12 17:15:20 PDT 2024</td>
</tr>
<tr><td>Total time:</td><td>127 seconds (127342 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.loginpage.Forgottenemail.navigation()'><b>navigation</b><br>Test class: com.loginpage.Forgottenemail</td>
<td><div><pre>java.lang.AssertionError: An unexpected error occurred and page is blank
	at com.loginpage.Forgottenemail.navigation(Forgottenemail.java:121)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//p[normalize-space()=&apos;Welcome, Login to your account.&apos;]&quot;}
  (Session info: chrome=126.0.6478.127)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &apos;4.22.0&apos;, revision: &apos;c5f3146703*&apos;
System info: os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;21.0.3&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f90084fe04503a4bfd72ae5b9f5587d8, findElement {using=xpath, value=//p[normalize-space()=&apos;Welcome, Login to your account.&apos;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.127, chrome: {chromedriverVersion: 126.0.6478.126 (d36ace6122e..., userDataDir: C:\Users\Jerin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50801}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50801/devtoo..., se:cdpVersion: 126.0.6478.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: f90084fe04503a4bfd72ae5b9f5587d8
	at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
	at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
	at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:349)
	at com.loginpage.Forgottenemail.navigation(Forgottenemail.java:101)
	... 26 more
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace50345623", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace50345623'><pre>java.lang.AssertionError: An unexpected error occurred and page is blank
	at org.testng.Assert.fail(Assert.java:87)
	at com.loginpage.Forgottenemail.navigation(Forgottenemail.java:121)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//p[normalize-space()=&apos;Welcome, Login to your account.&apos;]&quot;}
  (Session info: chrome=126.0.6478.127)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &apos;4.22.0&apos;, revision: &apos;c5f3146703*&apos;
System info: os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;21.0.3&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f90084fe04503a4bfd72ae5b9f5587d8, findElement {using=xpath, value=//p[normalize-space()=&apos;Welcome, Login to your account.&apos;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 126.0.6478.127, chrome: {chromedriverVersion: 126.0.6478.126 (d36ace6122e..., userDataDir: C:\Users\Jerin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50801}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50801/devtoo..., se:cdpVersion: 126.0.6478.127, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: f90084fe04503a4bfd72ae5b9f5587d8
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
	at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
	at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:349)
	at com.loginpage.Forgottenemail.navigation(Forgottenemail.java:101)
	... 26 more
</pre></div></td>
<td>20</td>
<td>com.loginpage.Forgottenemail@54a7079e</td></tr>
<tr>
<td title='com.loginpage.Forgottenemail.existemailid()'><b>existemailid</b><br>Test class: com.loginpage.Forgottenemail</td>
<td><div><pre>java.lang.AssertionError: expected [Username not recognized. Please try again.] but found [User does not exists]
	at com.loginpage.Forgottenemail.existemailid(Forgottenemail.java:64)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace670084377", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace670084377'><pre>java.lang.AssertionError: expected [Username not recognized. Please try again.] but found [User does not exists]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:629)
	at org.testng.Assert.assertEquals(Assert.java:639)
	at com.loginpage.Forgottenemail.existemailid(Forgottenemail.java:64)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>4</td>
<td>com.loginpage.Forgottenemail@54a7079e</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.verifytheinvalidname()'><b>verifytheinvalidname</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td><div><pre>java.lang.AssertionError: The name field should not accept numbers. expected [false] but found [true]
	at com.loginpage.Validloginwithdashboard.verifytheinvalidname(Validloginwithdashboard.java:249)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace399699227", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace399699227'><pre>java.lang.AssertionError: The name field should not accept numbers. expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertFalse(Assert.java:67)
	at com.loginpage.Validloginwithdashboard.verifytheinvalidname(Validloginwithdashboard.java:249)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>2</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Forgottenemail.validdata()'><b>validdata</b><br>Test class: com.loginpage.Forgottenemail</td>
<td><div><pre>java.lang.AssertionError: expected [A password reset link has been sent to your email address.] but found [Email has been sent...
The reset link has been sent to your email address]
	at com.loginpage.Forgottenemail.validdata(Forgottenemail.java:85)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace65438140", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace65438140'><pre>java.lang.AssertionError: expected [A password reset link has been sent to your email address.] but found [Email has been sent...
The reset link has been sent to your email address]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:629)
	at org.testng.Assert.assertEquals(Assert.java:639)
	at com.loginpage.Forgottenemail.validdata(Forgottenemail.java:85)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>4</td>
<td>com.loginpage.Forgottenemail@54a7079e</td></tr>
<tr>
<td title='com.loginpage.Loginpage.validation()'><b>validation</b><br>Test class: com.loginpage.Loginpage</td>
<td><div><pre>java.lang.AssertionError: expected [please eneter your email id ] but found [Fill email]
	at com.loginpage.Loginpage.validation(Loginpage.java:89)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1092619788", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1092619788'><pre>java.lang.AssertionError: expected [please eneter your email id ] but found [Fill email]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:629)
	at org.testng.Assert.assertEquals(Assert.java:639)
	at com.loginpage.Loginpage.validation(Loginpage.java:89)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>3</td>
<td>com.loginpage.Loginpage@71a8adcf</td></tr>
<tr>
<td title='com.loginpage.Loginpage.testBlankPassword()'><b>testBlankPassword</b><br>Test class: com.loginpage.Loginpage</td>
<td><div><pre>java.lang.AssertionError: Validation message does not match when password is blank! expected [please enter password] but found [Fill password]
	at com.loginpage.Loginpage.testBlankPassword(Loginpage.java:112)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace47918293", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace47918293'><pre>java.lang.AssertionError: Validation message does not match when password is blank! expected [please enter password] but found [Fill password]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertEqualsImpl(Assert.java:140)
	at org.testng.Assert.assertEquals(Assert.java:122)
	at org.testng.Assert.assertEquals(Assert.java:629)
	at com.loginpage.Loginpage.testBlankPassword(Loginpage.java:112)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>7</td>
<td>com.loginpage.Loginpage@71a8adcf</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.validationmessageofsalesfollowup()'><b>validationmessageofsalesfollowup</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.validationmessageofcontactinfo()'><b>validationmessageofcontactinfo</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.verifythedropdown()'><b>verifythedropdown</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Loginpage.logincredentails()'><b>logincredentails</b><br>Test class: com.loginpage.Loginpage</td>
<td></td>
<td>2</td>
<td>com.loginpage.Loginpage@71a8adcf</td></tr>
<tr>
<td title='com.loginpage.Forgottenemail.forgotpassword()'><b>forgotpassword</b><br>Test class: com.loginpage.Forgottenemail</td>
<td></td>
<td>9</td>
<td>com.loginpage.Forgottenemail@54a7079e</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.manadatorysymbolofbranch()'><b>manadatorysymbolofbranch</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.verifyname()'><b>verifyname</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>2</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Loginpage.testButtonIsEnabled()'><b>testButtonIsEnabled</b><br>Test class: com.loginpage.Loginpage</td>
<td></td>
<td>0</td>
<td>com.loginpage.Loginpage@71a8adcf</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.validcrendentials()'><b>validcrendentials</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>6</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.mandatorysymbolofphonenumberfield()'><b>mandatorysymbolofphonenumberfield</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.mandatorysymbolofname()'><b>mandatorysymbolofname</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.crm()'><b>crm</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.mandotrysymboloftype()'><b>mandotrysymboloftype</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.verifycheckbox()'><b>verifycheckbox</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Loginpage.invalidlogin()'><b>invalidlogin</b><br>Test class: com.loginpage.Loginpage</td>
<td></td>
<td>0</td>
<td>com.loginpage.Loginpage@71a8adcf</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.lead()'><b>lead</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>1</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.contactinfo()'><b>contactinfo</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>20</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
<tr>
<td title='com.loginpage.Validloginwithdashboard.mandatorysymbolofemail()'><b>mandatorysymbolofemail</b><br>Test class: com.loginpage.Validloginwithdashboard</td>
<td></td>
<td>0</td>
<td>com.loginpage.Validloginwithdashboard@26e356f0</td></tr>
</table><p>
</body>
</html>